packages/loot-core/src/platform/server/fs/index.electron.ts

```diff
@@ -10,10 +10,16 @@ export { getDocumentDir, getBudgetDir, _setDocumentDir } from './shared';
 
 let rootPath = path.join(__dirname, '..', '..', '..', '..');
 
-if (__filename.match('bundle')) {
-  // The file name is not our filename and indicates that we're in the
-  // bundled form. Because of this, the root path is different.
-  rootPath = path.join(__dirname, '..');
+switch (__filename) {
+  case 'bundle.api.js': // api bundle uses the electron bundle - account for its file structure
+    rootPath = path.join(__dirname, '..');
+    break;
+  case 'bundle.desktop.js': // electron app
+    rootPath = path.join(__dirname, '..', '..');
+    break;
+  default:
+    rootPath = path.join(__dirname, '..', '..');
+    break;
```

Comment:
I discovered the API uses the electron bundle (not sure why). This was working by accident - the api bundle was one directory deep and so was the electron bundle (at the time).

I'm being more specific here so it makes more sense.